test/functional/rpc_net.py

```diff
@@ -102,5 +111,40 @@ def _test_getpeerinfo(self):
         assert_equal(peer_info[0][0]['addrbind'], peer_info[1][0]['addr'])
         assert_equal(peer_info[1][0]['addrbind'], peer_info[0][0]['addr'])
 
+    def _test_getnodeaddresses(self):
+        self.nodes[0].add_p2p_connection(P2PInterface())
+
+        # send some addresses to the node via the p2p message addr
+        msg = msg_addr()
+        imported_addrs = []
+        for i in range(256):
+            a = "123.123.123.{}".format(i)
+            imported_addrs.append(a)
+            addr = CAddress()
+            addr.time = 100000000
+            addr.nServices = NODE_NETWORK
+            addr.ip = a
+            addr.port = 8333
```

Comment:
This should probably be dashified right? @UdjinM6

Reply:
Regular nodes can have ports other than the default one (unlike MNs), so technically you can run a node on port 8333 and it shouldn't matter here.

Reply:
Yeah, it doesn't REALLY matter, but feels better to use the default port, but I guess it doesn't really matter.